CHANGELOG.md

```diff
@@ -1,3 +1,31 @@
+## v1.12.0 (30/08/2021)
+
+**olp-sdk-core**
+
+- Updated the `@here/olp-sdk-dataservice-api` dependency.
+
+**olp-sdk-authentication**
+
+- Removed the deprecated code (https://github.com/heremaps/here-data-sdk-typescript/pull/485).
```

Comment:
Link can be replaced with #485

Reply:
I've added it, because someone can read that in his editor, not on github. Or it's not a good idea?